### DISCUSSION TITLE
Non mi fa firmare il registro con la materia studio assistito

### DISCUSSION THREAD

**Discussion by CobraOfficial01** [iisgiua/giuaschool#766] [Richieste di aiuto] [answered]
![IMG_1154](https://github.com/user-attachments/assets/50e4e14e-0c20-4cc6-9128-78060e0dba00)

**Comment by trinko** [comment]
Dipende dal tipo di materia: devi controllare la materia che risulta già con la firma e vedere se il tipo è compatibile.
Es. di situazione non ammessa: in una classe senza configurazione dei gruppi, è presente una firma per la materia X e si vuole firmare con la materia Y che non appartiene al tipo SOSTEGNO.

**Reply by CobraOfficial01** [reply]
L’altro professore ha firmato per religione ed io che faccio studio assistito in un altra una non mi fa firmare

**Reply by trinko** [reply, marked as answer]
> L’altro professore ha firmato per religione ed io che faccio studio assistito in un altra una non mi fa firmare

Se ho capito il problema, puoi firmare come SUPPLENZA scegliendo il gruppo N.A. (non si avvalgono della religione).
